app/server.go

```diff
@@ -1790,17 +1819,24 @@ func (s *Server) doLicenseExpirationCheck() {
 		return
 	}
 
-	if !license.IsPastGracePeriod() {
-		mlog.Debug("License is not past the grace period.")
-		return
-	}
-
 	users, err := s.Store.User().GetSystemAdminProfiles()
 	if err != nil {
 		mlog.Error("Failed to get system admins for license expired message from Mattermost.")
 		return
 	}
 
+	if license.IsAroundSixtyDaysToExpiration() {
```

Comment:
Genericize this function name a bit more. 

`IsWithinExpirationPeriod` or something along those lines.